README.md

```diff
@@ -504,7 +541,7 @@ evaluation doesn't have to be run on each set separately.
 
 If you use this work, please cite [our paper](https://arxiv.org/abs/2103.16544) (to appear in CVPR'21):
 
-```python
+```bibtex
```

Comment:
Interesting! I didn't know GitHub supported `bibtex` formatting, since it's not supported on nearly all JavaScript code highlighters.

Reply:
Yah, I was also surprised it worked :)!